docs/generated/manifests/menus.json

```diff
@@ -2684,6 +2684,14 @@
             "isExternal": false,
             "children": [],
             "disableCollapsible": false
+          },
+          {
+            "name": "Identify Dependencies Between Folders",
+            "path": "/recipes/other/identify-dependencies-between-folders",
+            "id": "identify-dependencies-between-folders",
+            "isExternal": false,
+            "children": [],
+            "disableCollapsible": false
```

Comment:
These seem unrelated? maybe needs to be rebases with master?

Reply:
ah yeah, i can try that